tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AdoNet/ReaderCloseIntegration.cs

```diff
@@ -60,7 +49,6 @@ internal static CallTargetReturn OnMethodEnd<TTarget>(TTarget instance, Exceptio
                 }
             }
 
-            state.Scope.DisposeWithException(exception);
```

Comment:
don't we want to record the exception here and below ? Is it done automatically ?

Reply:
We are not generating spans so there's no need for that

Reply:
oh ok right